fspiop-api/documents/API-Definition_v1.1.1.md

```diff
@@ -2395,6 +2409,24 @@ If the server is unable to find or create a transfer, or another processing erro
 
 **Table 33 -- PUT /transfers/_{ID}_/error data model**
 
+#### 6.7.5.2 PATCH /transfers/_{ID}_/error
+
+Alternative URI: N/A
+
+Logical API service: **Notify Transfer Information Error**
+
+If the server is unable to commit a transfer, or another processing or validation error occurs, during the processing of a **PUT /transfers/**_{ID}_ callbac, the notification **PATCH**
+
```

Comment:
This newline should removed as the text should continue

Reply:
Removed